text/0000-additional-exported-layers.md

```diff
@@ -0,0 +1,123 @@
+# Meta
+[meta]: #meta
+- Name: Additional exportable layers
+- Start Date: 2021-08-13
+- Author(s): [@samj1912](https://github.com/samj1912)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Allow users to provide additional paths that can be exported similar to the application directory and allow easy configuration of the application directory. These paths are defined via the [`VOLUME`](https://docs.docker.com/engine/reference/builder/#volume) directive in `Dockerfile` and stored in the [`Volumes` object](https://github.com/opencontainers/image-spec/blob/main/config.md#properties) in the image config.
+
+# Definitions
+[definitions]: #definitions
+
+- `Workspace`: The default application directory where the application source code is mounted and final build outputs are written. This directory is exported.
+- `Volume`: A set of directories describing where the buildpack process is likely to write data specific and should be exported alongside the application workspace. 
+- `WorkingDir`: The default working directory for a container.
+
+# Motivation
+[motivation]: #motivation
+
+The main motivation behind this RFC is to unlock exporting application images that require libraries or software to be installed in certain directories apart from the default application directory and these installations are rebase safe and self-contained. This unlocks use cases where buildpacks may install software in `/opt` directories (for AWS Lambda extensions or other common standalone software) for example or preserve some settings in a `/home/$USER/.config` for the output application. This provides a rebase safe extension point without major changes to the buildpacks API.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Users would be able to define stack images with `VOLUME` directives in Dockerfiles.
+
+
+For example a build/run base image may look like - 
```

Comment:
Is it confusing to add these volumes to the run image, given that when we extend the run image to create the app image these directories will have buildpack-contributed contents and thus will not behave as volumes?

If did wanted these as volumes on the run image (to prevent folks from extending the run image in a way that breaks rebase) I think the exporter should remove these volumes from the final app image.